packages/react-components/src/components/styles/TypingIndicator.styles.ts

```diff
@@ -7,7 +7,7 @@ import { mergeStyles } from '@fluentui/react';
  * @private
  */
 export const typingIndicatorContainerStyle = mergeStyles({
-  minHeight: '2.125rem'
+  minHeight: '0.125rem'
```

Comment:
The min-height is actually there to prevent the messages from shifting up and down when new messages appear (this should have a comment to make that clear).

e.g. when a typing indicator appears and dissapears is causes the messages to shift up and down:
![image](https://user-images.githubusercontent.com/2684369/149187022-810cfb28-c44c-4913-b56e-684a8209af3a.png)

Is it possible to keep the minHeight and instead align the text (vertically) to the bottom or center of the container so it is not so far away from the edit box?